ethcore/src/spec.rs

```diff
@@ -58,6 +58,8 @@ pub struct Spec {
 
 	/// Known nodes on the network in enode format.
 	pub nodes: Vec<String>,
+	/// Network ID
+	pub network_id: U256,
```

Comment:
Well, I've added a number of configuration options in this PR, and that's one of them.